src/cloud/validateCode.js

```diff
@@ -61,7 +61,7 @@ const validateCode = async request => {
     if (!sessionToken) {
       const logged = await Parse.User.logIn(
         user.getUsername(),
-        generatePassword(installationId),
```

Comment:
So do I not need to pass up the installationId anymore?

Reply:
It's not necessary to pass up on the request params. Parse will send automatically if It has It

Reply:
Ok Ill remove in from the validate code. Do we still need it for sendCode?

Reply:
Nope, you can remove it